script.js

```diff
@@ -20,23 +23,29 @@ function checkEnd(penguinPosition) {
     return penguinPosition > 1000
 }
 
-function endGame() {
+function endGameMessage() {
     alert('Far out brah! That was totally radical, you found some sweet lines down those faces! Catch you on the flip')
 }
 
 startButton.addEventListener('click', () => {
     splashScreen.style.display = "none"
-    document.addEventListener("keydown", (event) => {
-        if (event.code === "ArrowLeft" || event.code === "ArrowRight") {
-            if (checkKey(lastKeyPressed, event.code)) {
-                lastKeyPressed = event.code
+    document.addEventListener("keydown", function playGame(e) {
+        if (lastKeyPressed === '') {
+            raceStopwatch = setInterval(() => {
+                timer++
+                gameTimerDisplay.textContent = timer / 10
+            }, 100)
```

Comment:
this is now incrementing the timer every 10th of a second, which is better, however:
1. i would like it to be every 100th of a second
2. it would be better if the `timer` variable itself is in seconds, so that in future if you have a leaderboard you can display the times without performing any calculations first